repository_service_tuf/cli/artifact/repository.py

```diff
@@ -0,0 +1,261 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+
+import base64
+
+from click import Context
+from dynaconf.loaders.yaml_loader import write  # type: ignore
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+
+
+def write_config(settings_path, settings_data, merge=True):
+    write(settings_path, settings_data, merge)  # pragma: no cover
+
+
+@artifact.group()
+@click.pass_context
+def repository(context) -> None:
+    """
+    Repository management.
+    """
+
+
+@repository.command()
+@click.option(
+    "-a",
+    "--all",
+    help="Show all the configured repositories.",
+    is_flag=True,
+    required=None,
+)
+@click.pass_context
+@click.argument("repository", required=False)
+def show(
+    context: Context,
+    all: bool,
+    repository,
```

Comment:
repository: string
```suggestion
    repository: str,
```

Reply:
Same here - if we agree to change the initial layout, I'll update it.